### DISCUSSION TITLE
How to use the command system

### DISCUSSION THREAD

**Discussion by NotAJocke** [tesselode/kira#108] [Q&A] [answered]
I've read the documentation, I don't understand what to do with a Parameter.

I'm trying to update the manager's volume from another thread

Thanks in advance

**Comment by tesselode** [comment, marked as answer]
You can wrap handles in `Arc<Mutex<_>>` if you need to share them across threads. You don't necessarily need a `Parameter`; you can change the volume of the main mixer track (which will affect the volume of all audio) using [`AudioManager::main_track`](https://docs.rs/kira/latest/kira/manager/struct.AudioManager.html#method.main_track) and [`TrackHandle::set_volume`](https://docs.rs/kira/latest/kira/track/struct.TrackHandle.html).
